pkg/migrator/migrator.go

```diff
@@ -13,8 +13,8 @@ import (
 )
 
 const (
-	//CscCRDName is the CatalogSourceConfig CRD name
-	CscCRDName = "catalogsourceconfigs.operators.coreos.com"
+	//OpsrcCRDName is the OperatorSource CRD name
+	OpsrcCRDName = "operatorsources.operators.coreos.com"
```

Comment:
Is there ever a case where we need to migrate away from both `CatalogSourceConfig` _and_ `OperatorSource` in a single upgrade?

Reply:
That is a good point. Do we allow migration from suppose 4.3 to 4.6 directly? Or do users have to follow the migration chain, i.e 4.3->4.4->4.5->4.6 ?

Reply:
I don't believe that we allow skipping minor OpenShift versions, but we should check before merging this as customers may be using CSCs or OpSrcs and this change will result in data loss.

Reply:
You can't skip minor versions